src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -483,12 +483,13 @@ public void buffer(SinkRecord sinkRecord) {
   }
 
   /**
-   * Get the offset that was last committed to HDFS, or null if nothing was committed yet.
-   *
-   * @return the last committed offset; may be null
+   * HDFS Connector tracks offsets in filenames in HDFS (for Exactly Once Semantics) as the last
+   * record's offset that was written to the last file in HDFS.
+   * This method returns the next offset after the last one in HDFS, useful for some APIs
+   * (like Kafka Consumer offset tracking).
```

Comment:
This is not a well-formed JavaDoc (missing `@return`), and it should also mention when this method return `-1`.